app/uk/gov/hmrc/gform/sharedmodel/formtemplate/EmailParameter.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.sharedmodel.formtemplate
+
+import play.api.libs.json.{ Json, OFormat }
+case class EmailParameter(emailTemplateVariable: String, value: String)
```

Comment:
The variables in the email parameters are of the form ${xxx}, you need to check that is true and strip out the ${} .

It would also be a good idea to check when the template is parsed that the field xxx actually exists.  Otherwise no problem will be found until run time & even then only in QA or Production where email sending works

Reply:
👍 for checking that fields actually exists, but I'm not sure we want simple strip out `${}`. Shouldn't we treat `value` as full blown expression (`Expr`)?